### ISSUE TITLE
'focusableTarget is null' when clicking Date Combo Box

### ISSUE THREAD

**Issue by zenwork** [elix/elix#91]
I am trying to use the Date Combo Box in a webcomponent project based on lit-element written in Typescript. The combo loads and runs correctly except for this error 'focusableTarget is null' when I click anywhere in the flyout. the actions are correctly captured before it closes. For example clicking on the right chevron will move the calendar forward by one month before the flyout closes.

here is my render fuction
```
protected render(): TemplateResult {
    return html`
            <h1>Elix Example</h1>

            <elix-date-combo-box
            year-format="2-digit"
            month-format="2-digit"
            days-of-week-format="long"
            >
          `;
}
```

Am I doing something wrong?

**Comment by JanMiksovsky** [comment]
I'm having trouble reproducing the error message you describe. I've created a local page similar to the one at https://component.kitchen/demos/dateComboBox.html, but I don't see the error.

That variable name, `focusableTarget`, comes up in a utility called `forwardFocus` that I updated a few weeks ago. I believe I saw a problem similar to the one you're seeing. It's a little frustrating to me that I can't reproduce your problem, but there's a chance the problem will fixed in our imminent Elix 6.0 release. When 6.0 is published, I hope I can ask you to try to reproduce the bug in that release.

In the meantime, if you have a way to share a live repro case (via a site like jsBin, jsFiddle, CodePen, etc.) that'd be helpful. If that's too much trouble, I understand.

Some other notes:
* Normally you'd want to explicitly close a web component tag: `<elix-date-combo-box></elix-date-combo-box>`. Browsers will try to close unclosed tags for you, but the results can be unpredictable. I'd be surprised if that's an issue in this case, however.
* Your sample code makes the completely reasonable assumption that the various `CalendarMonth` formatting options are exposed on `DateComboBox`. But that doesn't appear to be the case: `year-format`, etc., are currently *not* exposed on `DateComboBox`. That's an oversight! I'll fix that.

**Comment by JanMiksovsky** [comment]
If you want to try this against the pending 6.0 release, I've published a beta package. You should be able to pick it up by referencing:

```json
"dependencies": {
  "elix": "6.0.0-beta.0"
}
```

**Comment by zenwork** [comment]
here is a simple parcel-based project that shows the bug I am seeing witht he combo.
[x-elix.zip](https://github.com/elix/elix/files/3217817/x-elix.zip)

**Comment by JanMiksovsky** [comment]
This should now be fixed in the just-released 6.0.0.

**Comment by zenwork** [comment]
thank you.
